e2e_test/batch/functions/session_timezone.slt

```diff
@@ -55,5 +55,35 @@ select '2022-01-01 00:00:00-08:00'::timestamp with time zone::varchar;
 ----
 2022-01-01 00:00:00-08:00
 
+statement ok
+set timezone = 'europe/london'
+
+# Add/Subtract timestamptz with interval across a daylight savings boundary
+
+# This should first add the days (naively), then the hours, with timezone in mind.
+# Daylight savings falls on 2016-10-30 in London timezone
+query T
+select '2016-10-29 12:00:00'::timestamptz + interval '24 hours' + interval '1 day';
+----
+2016-10-31 11:00:00+00:00
+
+# This should first add the days (naively), then the hours, with timezone in mind.
```

Comment:
Are these 2 test cases supposed to have the same comment?

Reply:
No, they are different, depending on the order of operation, as the `interval '24 hours'` meaning will change at the daylight saving boundary

Reply:
So one of the comment is wrong and needs a change, right?

Actually I think it is the meaning of `1 day` changed to mean `25 hours` during winter backward adjustment.

Reply:
Yes, sry, I've changed it. I'll be even more explicit to be clearer.